### PR TITLE
[MDB IGNORE] Gets rid of the /brig shuttle floor subtype

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

Removes the /brig red plastitanium shuttle floor, replaces the checks related to it with checking if the area is the shuttle brig
From what I checked in vscode, all the shuttles that use this floor already also have the shuttle brig area on the same tiles so it changes nothing (and might even fix things if shuttles don't use this floor type for their shuttle brig)

## Why It's Good For The Game

Why the fuck are we checking for a TURF

## Changelog

Not player facing

### CODE REVIEW

**Comment by Mothblocks** [comment]
FYI This change has been proposed several times but never approved because the upside to having it be based on tile is that it is always unambiguous if you are failing your objective or not.

**Comment by SgtHunk** [comment]
The PR that introduced the area-based escape was https://github.com/tgstation/tgstation/pull/55677, but it only changed it for the escape objective while the other two changes, `elimination_hijack()` and `intoTheSunset()` still checked for the brig plastitanium tiles.
Making them area-based instead of checking for turf seems a lot better on the long run, considering antags should be checking if they're brigged (aka if they're in the shuttle brig area) rather than if they're standing on a specific tile because there's no certainty that the turf they're on IS actually the shuttle brig other than trusting that the mapper used the tiles, and forcing mappers to stick to a certain type of brig tiles limits creativity.

Shuttles that had brig floors outside the shuttle brig include the basketball shuttle (https://github.com/tgstation/tgstation/pull/67061) and the medisim
Shuttles that had shuttle brigs without brig floors include the shadow shuttle (https://github.com/tgstation/tgstation/pull/70626) and zeta

**Comment by Mothblocks** [comment]
If other stuff already uses an area then yeah go ahead, just answering the question in your "why is this good for the game"

File: code/game/turfs/open/floor/mineral_floor.dm [hunk 1]
```diff
@@ -174,9 +174,6 @@
 /turf/open/floor/mineral/plastitanium/red/snow_cabin
 	temperature = 180
 
-/turf/open/floor/mineral/plastitanium/red/brig
```
Comment (san7890) [inline code/game/turfs/open/floor/mineral_floor.dm:177]: make an updatepaths for this change https://github.com/tgstation/tgstation/tree/master/tools/UpdatePaths

**Comment by san7890** [comment]
i am slightly concerned about the possible desync between user feedback and what constitutes the "brig area" but i don't think there's ever going to be an opportunity to mistake a deliberately designed brig area on any given shuttle for some other type of room IMO

**Pull request merged**